config/jobs/kubernetes-sigs/testing_frameworks/testing-frameworks-config.yaml

```diff
@@ -5,15 +5,32 @@ presubmits:
     - master
     always_run: true
     skip_report: false
+    decorate: true
     labels:
       preset-service-account: "true"
+      preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
-        args:
-        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
-        - "--root=/go/src"
-        - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - --scenario=execute
-        - --
+      - command:
```

Comment:
You'll need to provide an image that has all the dependencies necessary to run ./bin/test-on-prow.sh (aka anything the test will use that isn't installed by this script, such as docker and golang).